ws-client/src/transport.rs

```diff
@@ -24,11 +24,13 @@
 // IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
+use crate::stream::EitherStream;
 use crate::tokio::{TcpStream, TlsStream};
 use futures::io::{BufReader, BufWriter};
 use futures::prelude::*;
 use soketto::connection;
 use soketto::handshake::client::{Client as WsRawClient, ServerResponse};
```

Comment:
Can we use `SokettoClient` rather than `WsRawClient`?

Reply:
renamed it WsHandshakeClient I think it's more descriptive...